lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -11,17 +11,14 @@ import {
 
 import { ApiError, apiCall } from '../utils/api';
 import { Config } from '../config';
+import { ClientRpc, useService } from '../services';
```

Comment:
Is this the wrong time to mention the casing convention that would suggest this should be `ClientRPC`?

Reply:
Indeed. It turns out that requires updates in quite a few places so I hope you don't mind if I leave that to a follow-up PR.